utils/misc.py

```diff
@@ -50,50 +50,99 @@ def export_points_to_ply(
-            aabb_min, aabb_max = aabb[:3], aabb[3:]
-            aabb_center = (aabb_min + aabb_max) / 2
-            aabb_sacle_max = (aabb_max - aabb_min).max() / 2 * 1.1
-            vis_mask = torch.logical_and(positions >= aabb_min, positions < aabb_max).all(-1)
-        else:
-            aabb_center = positions.mean(0)
-            aabb_sacle_max = (positions - aabb_center).abs().max() * 1.1
-            vis_mask = torch.ones_like(positions[:, 0], dtype=torch.bool)
-            
-        positions = ((positions[vis_mask] - aabb_center) / aabb_sacle_max).cpu().numpy()
-        map_to_tensors["positions"] = o3d.core.Tensor(positions, o3d.core.float32)
-        map_to_tensors["normals"] = o3d.core.Tensor(np.zeros_like(positions), o3d.core.float32)
-
-        colors = model.colors[vis_mask].data.cpu().numpy()
-        map_to_tensors["colors"] = (colors * 255).astype(np.uint8)
+        positions = model.means 
+        # if aabb is not None:
+        #     aabb = aabb.to(positions.device)
+        #     aabb_min, aabb_max = aabb[:3], aabb[3:]
+        #     aabb_center = (aabb_min + aabb_max) / 2
+        #     aabb_scale_max = (aabb_max - aabb_min).max() / 2 * 1.1
+        #     vis_mask = torch.logical_and(positions >= aabb_min, positions < aabb_max).all(-1)
+        # else:
+        #     aabb_center = positions.mean(0)
+        #     aabb_scale_max = (positions - aabb_center).abs().max() * 1.1
+        #     vis_mask = torch.ones_like(positions[:, 0], dtype=torch.bool)
+
+        # positions = ((positions[vis_mask] - aabb_center) / aabb_scale_max).cpu().numpy()
+        vis_mask = torch.ones_like(positions[:, 0], dtype=torch.bool)
+        positions = positions[vis_mask].cpu().numpy()
+        positions = positions
+
+        normals = np.zeros_like(positions)  # normals: 0
+
+        # colors = model.colors[vis_mask].data.cpu().numpy()
+        colors = model.shs_0[vis_mask].data.cpu().numpy()
         for i in range(colors.shape[1]):
-            map_to_tensors[f"f_dc_{i}"] = colors[:, i : i + 1]
-
-        shs = model.shs_rest[vis_mask].data.cpu().numpy()
-        if model.config.sh_degree > 0:
-            shs = shs.reshape((colors.shape[0], -1, 1))
-            for i in range(shs.shape[-1]):
-                map_to_tensors[f"f_rest_{i}"] = shs[:, i]
-
-        map_to_tensors["opacity"] = model.opacities[vis_mask].data.cpu().numpy()
+            pass 
+
+        shs = model.shs_rest[vis_mask].data.cpu().numpy() 
+        shs = shs.reshape((colors.shape[0], -1))
+
+        # opacity = model.get_opacity[vis_mask].data.cpu().numpy() 
+        opacity = model.pure_opacity[vis_mask].data.cpu().numpy() 
+        # scales = model.get_scaling[vis_mask].data.cpu().numpy()
+        scales = model.pure_scaling[vis_mask].data.cpu().numpy()
+        # quats = model.get_quats[vis_mask].data.cpu().numpy()
+        quats = model.pure_quats[vis_mask].data.cpu().numpy()
+        num_points = positions.shape[0]
```

Comment:
Thanks!

As suggested in the first comment, I suggest just using `model._means` `model._quats` ... here